src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -970,12 +954,20 @@ static void CheckForOldValueGenerationAnnotation([NotNull] IAnnotatable annotata
                 .Append("ALTER ")
                 .Append(type)
                 .Append(' ')
-                .Append(name)
+                .Append(Dependencies.SqlGenerationHelper.DelimitIdentifier(name, schema))
                 .Append(" RENAME TO ")
-                .Append(newName)
-                .AppendLine(Dependencies.SqlGenerationHelper.StatementTerminator);
+                .Append(Dependencies.SqlGenerationHelper.DelimitIdentifier(newName))
+                .AppendLine(';');
```

Comment:
Is this is the only diff with 266aa00279c88c8e568a48e5936bfb962f21ced7? 

If so, we may also need to pull in 161eebc88fdab9bf14c00214d8e6b2fe8c07964d, which seems to be the source of this semicolon on the `2.2.x` and `dev` branches.

Reply:
Scratch that. Just replace the semicolon literal with `Dependencies.SqlGenerationHelper.StatementTerminator` to mirror the rest of the methods in this branch.

Reply:
Done

Reply:
Can you fix this literal too? I know it's divergent from the version in `dev`, but since the rest of the class still uses the dependency terminator, we might as well match it as best we can.

Reply:
Done. There are 2 more, lines 408 and 714, should I change those too?